Profile Guided Optimization (PGO) not working on aarch64

Using the latest nightly on `aarch64-unknown-linux-gnu` installed with `rustup`, I am getting the following error when I attempt to run any crate with PGO:

```
$ RUSTFLAGS="-Z pgo-gen=/tmp/prof.data" cargo run                                                                   Compiling play v0.1.0 (/home/me/play)
error[E0463]: can't find crate for `profiler_builtins`                                                                  
error: aborting due to previous error
```

--- a/src/ci/docker/dist-aarch64-linux/Dockerfile
+++ b/src/ci/docker/dist-aarch64-linux/Dockerfile
@@ -32,5 +32,8 @@ ENV CC_aarch64_unknown_linux_gnu=aarch64-unknown-linux-gnueabi-gcc \
 
 ENV HOSTS=aarch64-unknown-linux-gnu
 
-ENV RUST_CONFIGURE_ARGS --enable-extended --disable-docs
+ENV RUST_CONFIGURE_ARGS \
+      --enable-extended \
+      --enable-profiler \
+      --disable-docs
 ENV SCRIPT python2.7 ../x.py dist --host $HOSTS --target $HOSTS

